config/areas/account.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
```

Comment:
For better performance the `Panel` class should be explicitly imported:

```suggestion

use Kirby\Panel\Panel;

```